Objects/fileobject.c

```diff
@@ -239,7 +239,7 @@ _PyLong_FileDescriptor_Converter(PyObject *o, void *ptr)
 ** identically.
 */
 char *
-Py_UniversalNewlineFgets(char *buf, int n, FILE *stream, PyObject *fobj)
+_Py_UniversalNewlineFgetsWithSize(char *buf, int n, FILE *stream, PyObject *fobj, size_t* size)
```

Comment:
Nit: Maybe move the docstring down to the new definition of `Py_UniversalNewlineFgets`?

Reply:
Makes sense, will do